app/controllers/application_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```

Comment:
- [ ] Right now I am not being able to sign up to use the app, because an error is being raised saying that "name can't be blank", but I am filling the form properly, here's an image from console about the error itself:
![Captura de pantalla_2022-05-20_07-07-10](https://user-images.githubusercontent.com/59100477/169454774-1cfa387a-aee5-44b8-a5a7-a7111e77c6db.png)
So please work on this issue.